.github/workflows/ci-build-macos.yaml

```diff
@@ -39,6 +39,13 @@ jobs:
     timeout-minutes: 120
 
     steps:
+      - name: cancle previours runs
+        uses: apache/airflow-cancel-workflow-runs@953e057dc81d3458935a18d1184c386b0f6b5738
+        with:
+          cancelMode: allDuplicates
+          token: ${{ secrets.GITHUB_TOKEN }}
+          sourceRunId: ${{ github.event.workflow_run.id }}
```

Comment:
`sourceRunId` should be omitted when used in the same workflow which should be cancelled. [Documentation says about sourceRunId](https://github.com/apache/airflow-cancel-workflow-runs#inputs-and-outputs): "Useful only in workflow_run triggered events."